static/js/reducers/auth.js

```diff
@@ -29,15 +29,21 @@ export const getAuthPartialTokenSelector = R.path([
   "partial_token"
 ])
 export const getAuthFlowSelector = R.path(["auth", "data", "flow"])
-export const getAuthStateSelector = R.path(["auth", "data", "state"])
 export const getAuthProviderSelector = R.path(["auth", "data", "provider"])
-export const getAuthEmailSelector = R.path(["auth", "data", "email"])
 export const getFormErrorSelector = R.compose(
   R.head,
   R.pathOr([], ["auth", "data", "errors"])
 )
 export const isProcessing = R.path(["auth", "processing"])
 
+export const getAuthUiNameSelector = R.path(["ui", "authUserDetail", "name"])
```

Comment:
Could you move these to `ui.js`? they should live alongside the reducer they are for.